tests/tests.rs

```diff
@@ -1508,11 +1547,6 @@ fn test_exec_batch_with_limit() {
 
     let te = TestEnv::new(DEFAULT_DIRS, DEFAULT_FILES);
 
-    te.assert_output(
-        &["foo", "--batch-size", "0", "--exec-batch", "echo", "{}"],
-        "./a.foo ./one/b.foo ./one/two/C.Foo2 ./one/two/c.foo ./one/two/three/d.foo ./one/two/three/directory_foo",
-    );
```

Comment:
Why was this removed? Because it could fail on a hypothetical platform with a very low argmax limit?

Reply:
Ah no, because it makes assumptions about the sort order. Ok. I guess it's not really worth the effort. This path is already tested sufficiently in other tests.